examples/esbuild/WORKSPACE

```diff
@@ -25,12 +25,12 @@ http_archive(
     urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/4.0.0/rules_nodejs-4.0.0.tar.gz"],
 )
 
-load("@build_bazel_rules_nodejs//:index.bzl", "npm_install")
+load("@build_bazel_rules_nodejs//:index.bzl", "yarn_install")
 
-npm_install(
+yarn_install(
```

Comment:
why switch to yarn? PRs should be minimal to make them easy to review. If you're making a change that looks orthogonal, either put in a separate PR, or anticipate my question and add a comment about why this change is required for your change

Reply:
oh, thats because npm ignores the `resolutions` object in the package.json file. so we needed to install the dependencies with yarn_install just like other examples. but if this is a deal-breaker, I will try to find a way to make it work for both yarn_install and npm_install